### DISCUSSION TITLE
Renovate not updating Eclipse Temurin docker image

### DISCUSSION THREAD

**Discussion by jfisbein** [renovatebot/renovate#26963] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.109.4

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

I have a bunch of repos with Dockerfile files using `eclipse-temurin` docker image, most of them are pointing to version `21.0.1_12-jdk-jammy`, a new version `eclipse-temurin:21.0.2_13-jdk-jammy` has been published 5 days ago and it's [available at dockerhub](https://hub.docker.com/_/eclipse-temurin/tags?page=1&name=21.0.2_13-jdk-jammy), but renovatebot is not detecting it.

To test the _issue_, I created a repo with 2 simple files:
`Dockerfile` with content:
```Docker
FROM eclipse-temurin:21.0.1_12-jdk-jammy@sha256:ad942b12a8c5698da03ba8663312173cfc678e7eec12ea62abebdc54221b0e9f
```
`renovate.json` with content:
```json
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": [
      "config:recommended"
    ]
}  
```

Then I run renovatebot over the repository, and the dependency is found (see screenshot) but no update is proposed.

In this gist, you can find [renovatebot debug log](https://gist.github.com/jfisbein/e561b6386d5fb041fba1041cb64c1783). I checked and everything looks correct to me.

I'm pretty sure this worked in the past, last time I updated renovatebot was 4 months ago, and I created this very simple test repo to avoid any rule interference.

Does anyone know what I'm doing wrong?
![Screenshot Gitlab Issue](https://github.com/renovatebot/renovate/assets/272565/938535e8-7985-4778-bc6e-cf2ae7014945)

P.S: We use renovate for hundreds of projects, with PyPI, Maven, Gradle, npm, and everything is working, the only issue is with docker versions.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
This preset applies a custom versioning which is perhaps not compatible with this tagging: https://github.com/renovatebot/renovate/blob/c7ab3ca33711ab6783c7a5435230a8761b709a99/lib/config/presets/internal/workarounds.ts#L105-L130

**Reply by jfisbein** [reply, marked as answer]
If I understand correctly the `packageRules`, what it's trying to do is to filter the updates to only propose Java LTS versions like 8, 11, 17, and 21 avoiding proposing Java versions like 13 or 19.

In any case my problem solved when I updated Renovatebot from 36.x to 37.x 🤷🏼

**Reply by viceice** [reply]
you should always try latest renovate version first, especially when you're so far behind.
